iml-agent/src/action_plugins/firewall_cmd.rs

```diff
@@ -11,6 +11,7 @@ static DUPLICATE_MESSAGE: &[u8] = b"Warning: ALREADY_ENABLED\n";
 async fn port_change(action: &str, port: u16, proto: String) -> Result<(), ImlAgentError> {
     Command::new("firewall-cmd")
         .arg(format!("--{}-port={}/{}", action, port, proto))
+        .kill_on_drop(true)
```

Comment:
I noticed on check_stonith the `kill_on_drop` came before the args. It probably doesn't matter though, right?

Reply:
doesn't matter it's a builder interface. If it mattered it would fail to compile :)